docs/agents/browser.md

```diff
@@ -0,0 +1,123 @@
+title: Browser SDK
+description: Ship browser metrics to Sematext Experience.
+
+Sematext [Browser SDK](https://github.com/sematext/browser-sdk) is an open-source framework used to ship web application metrics related to user interaction. Using [Browser SDK](https://github.com/sematext/browser-sdk) allows gathering metrics related to:
+ 
+ - Hard page loads for traditional web applications as well as the single-page applications
+ - Soft page loads generated via the `routeChange` command for the single-page applications
```

Comment:
the single ... => single ... (2 lines above)

Reply:
Done.